tests/teamcity/update-builds.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -o errexit # exit on first command with non-zero status
+set -o nounset
+
+mkdir -p /home/ubuntu/firefox-channels
```

Comment:
Good point. I'll make it a command line arg, then $HOME, then /home/ec2-user ;-)